common/build/eslint-config-fluid/strict-biome.js

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * "Strict" Biome-compatible eslint configuration.
```

Comment:
Can you add a section to the package README about this? It currently enumerates the other configs and explains when they should be used.